homeassistant/helpers/llm.py

```diff
@@ -65,6 +65,7 @@
 DEFAULT_INSTRUCTIONS_PROMPT = """You are a voice assistant for Home Assistant.
 Answer questions about the world truthfully.
 Answer in plain text. Keep it simple and to the point.
+User's language is: {{ llm_context.language }}
```

Comment:
I don't think it can be `None` in the first place.

But:
1. This is just the default prompt, which means the user can remove it.
2. It does not say "Respond in this language only". I would expect LLMs to be smart enough to ignore this when they clearly see the user speaking a different language.